### ISSUE TITLE
Preview Env jobs can fail waiting on unnecessary pods

### ISSUE THREAD

**Issue by liam-j-bennett** [gitpod-io/gitpod#12622]
From Anton: https://gitpod.slack.com/archives/C032A46PWR0/p1662128693010439

Essentially, after using the installer to deploy gitpod, we (correctly) wait for the pods within the namespace to be ready. However, the namespace is used for Workspaces, which can fail or can have long startup/termination periods. Therefore, it makes sense to exclude them from the ready check in general, as they are not required to run gitpod.

We would need a relevant pod exclusion in https://github.com/gitpod-io/gitpod/blob/07c34348d56591f9ed5d10c3dd3bb2029f2b5a2f/.werft/util/kubectl.ts#L359

**Comment by kylos101** [comment]
:wave: I'll create a new issue referencing this one :smile:
